ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseEventMapper.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import uk.gov.hmcts.cmc.ccd.domain.CaseEvent;
+import uk.gov.hmcts.cmc.domain.models.ClaimDocumentType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.*;
+import static uk.gov.hmcts.cmc.domain.models.ClaimDocumentType.*;
+
+public class CaseEventMapper {
+
+    private static final Map<ClaimDocumentType, CaseEvent> claimDocumentTypeMap;
+
+    static {
```

Comment:
Please remove the static block. just add these while declaration like new Hashmap(){  put();    }

Reply:
Done